system/assets/js/admin/presidecore/formFields.js

```diff
@@ -1,5 +1,6 @@
 ( function( $ ){
 
+	var ulrPrefix = $(".auto-slug").attr("data-ulrPrefix");
```

Comment:
I'd move this down to within the codeblock that starts on line 8. While not likely, there is a potential for multiple auto slug fields.

Also, we use these auto slugs for more than just pages in the tree, so we'd need a way to know to use this feature only when editing *pages*.